### PR TITLE
Upgrade Node.js from 8.4.0 to 8.9.1 LTS

### PR DESCRIPTION
**Pull request by romandev**
Node.js 8.9.1 released. It's LTS(Long Term Support) version.

ISSUE=#174

### CODE REVIEW

File: template/bacardi_cpp.njk [hunk 1]
```diff
@@ -23,10 +23,11 @@
 #include "{{interface.idl_dir_name}}/{{interface.name | snakecase}}_bridge.h"
 {% endfor %}
 
-void Init(Napi::Env env, Napi::Object exports, Napi::Object module) {
+Napi::Object Init(Napi::Env env, Napi::Object exports) {
```
Comment (nadongguri) [inline template/bacardi_cpp.njk:26]: Is it one of changes to Node.js 8.9.1? :D

**Comment by nadongguri** [comment]
LGTM

**Review by hwanseung — APPROVED** [review]
LGTM

File: template/bacardi_cpp.njk (hunk 1 above)
Comment (romandev) [reply template/bacardi_cpp.njk:26]: @nadongguri Yeah, this is a change of wrapper API.

**Pull request merged**